docs/src/app_dev/overview/index.rst

```diff
@@ -8,47 +8,65 @@
+.. TODO(#1156): link to https://docs.digitalasset.com/build/current/ instead of 3.4 when it is available
 
-You can find more information in the following sub-sections:
+* Learn to build Canton Network applications from the tutorials, how-tos, explanations, and reference documentation at
+  https://docs.digitalasset.com/build/3.3/
+* Browse the currently featured apps: https://sync.global/featured-apps/
+* Request your app to be featured: https://sync.global/featured-app-request/
 
-.. toctree::
+  * These requests are published on the `Tokenomics mailing list <https://lists.sync.global/g/tokenomics/topics>`__.
+    Browse that list for inspiration and to see what other apps are being built.
 
-   read-only-apps
-   daml-apps
+* Join the app development discussion in the `#gsf-global-synchronizer-appdev <https://app.slack.com/client/T03T53E10/C08FQRCRFUN>`__
+  Slack channel by sending a request to operations@sync.global for your Slack organization to be added to the channel.
 
 
-.. todo::
+RPC APIs Overview
+-----------------
 
-   - add overview over types of APIs and which apps need what
-   - for Daml apps
+When building an application for the Canton Network,
+you will typically integrate with some of the HTTP or gRPC APIs provided by the Global Synchronizer and Validator Nodes.
+Use the guidance below to learn which ones to use and how to use them.
 
-      - explain app provider and app user nodes
-      - refer to the TSA training for the in-depth explanation of building Daml apps
+* Use the :ref:`app_dev_ledger_api` to access the view of the ledger as seen by the parties hosted on a Validator Node and submit transaction to the ledger.
+* Use the :ref:`app_dev_scan_api` to access the view of the ledger and its infrastructure as seen by all SV Nodes.
+  Note that this view is the one visible to the DSO party and does not include any of the data that is private to the parties hosted on Validator Nodes.
+  Use the :ref:`app_dev_ledger_api` to access the data of parties hosted on the Validator Node.
+* Use the :ref:`app_dev_validator_api` to access higher-level functionality provided by the
+  Splice Validator App running alongside the Canton Participant node in a
+  Validator Node.
 
+See the :ref:`validator-network-diagram` for details on the components running as part of a Validator Node's and the APIs they provide.
 
-.. todo:: split into validator/wallet api, scan api, daml, ledger API
-.. todo:: add section on testing including spinning up localnet
-.. todo:: add section on deployment for app devs, e.g., DAR uploads
-.. todo::
 
-    Add overview of how to integrate with CC at the Daml level
+Splice Daml APIs Overview
+-------------------------
 
-    - use the token standard
-    - mention the `AppPaymentRequestFlow` as deprecated
-    - clearly mark splice subscription API as deprecated
+Splice defines Daml APIs that decouple different applications on the Canton Network.
```

Comment:
I find this sentence confusing:  "Splice defines Daml APIs that decouple different applications on the Canton Network."

My mental model is that "Splice includes interoperability standards that are intended to reduce coupling between applications.  These standards and reference implementations are provided as Daml models."

I find the decoupling is a secondary aspect that is an intended side-effect.